packages/react/src/components/FeatureSection/__stories__/FeatureSection.stories.js

```diff
@@ -0,0 +1,3 @@
+# Feature section
```

Comment:
Hey @IgnacioBecerra , this is throwing prettier errors.

Reply:
Looks like you swapped the `.js` and `.mdx` content, @IgnacioBecerra.

Reply:
Ah good eye, the script had the wrong file content when generating the react wrapper for these two files. Updated with the correct file contents.